drivers/start_date_dq/app/models/start_date_dq/report.rb

```diff
@@ -0,0 +1,78 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module StartDateDq
+  class Report
+    include ActionView::Helpers::NumberHelper
+    include ArelHelper
+    include Filter::FilterScopes
+
+    attr_reader :filter
+
+    def initialize(user_id, filter = nil)
+      @filter = filter || default_filter(user_id)
+    end
+
+    def default_filter(user_id)
+      Filters::FilterBase.new(user_id: user_id, start: '2018-01-01')
+    end
+
+    def self.viewable_by(user)
+      GrdaWarehouse::WarehouseReports::ReportDefinition.where(url: url).
+        viewable_by(user).exists?
+    end
+
+    def self.url
+      'start_date_dq/warehouse_reports/reports'
+    end
+
+    def title
+      'Approximate Start Date Data Quality'
+    end
+
+    def column_names
+      ['# Days Difference',
+       'DateToStreetESSH',
+       'Entry Date',
+       'Personal ID',
+       'Project',
+       'Project Type']
```

Comment:
```suggestion
      [
        '# Days Difference',
         'DateToStreetESSH',
         'Entry Date',
         'Personal ID',
         'Project',
         'Project Type',
       ]
```
Nothing wrong with what you have, but we've been bitten dozens of times by the  missing `,` at the end of the line.  Usually it rears it's head when you go to add another row and miss needing to add a comma to the row above.  Also, it makes git changes much smaller (adding a row would only change the row in question, not the one above.

Reply:
Sure thing. Can I turn on the RuboCop rule for this? https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Layout/MultilineArrayBraceLayout

Reply:
https://github.com/greenriver/hmis-warehouse/pull/2141 added and enforcement rule for this so it can be fixed automatically in the future.

Reply:
Ooh, I like that!